src/validation/mod.rs

```diff
@@ -1,16 +1,12 @@
 use std::fmt;
-use elements::{
-	BlockType, External, GlobalEntry, GlobalType, Internal, MemoryType,
-	Module, Opcode, ResizableLimits, TableType, ValueType, InitExpr
-};
+use std::collections::HashMap;
+use elements::{BlockType, External, GlobalEntry, GlobalType, Internal, MemoryType, Module, Opcode,
+			   ResizableLimits, TableType, ValueType, InitExpr, Type};
```

Comment:
wrong idents and format overall

Reply:
Please check [updated](https://github.com/paritytech/parity-wasm/pull/128/commits/6f9f2915cd08797fde8284e4b6742e92893efa9b) version.